.cbmc-batch/jobs/Makefile.common

```diff
@@ -0,0 +1,206 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHELL=/bin/bash
+
+default: report
+
+# if Makefile.local exists, use it. This provides a way to override the defaults
+sinclude Makefile.local
+
+################################################################
+# Define locations and binaries
+ABSTRACTIONS ?= ""
+BASEDIR ?= $(abspath ../../../..)
+BATCH ?= cbmc-batch
+
+CBMCFLAGS += \
+	--bounds-check \
+	--pointer-check \
+	--function harness \
+	$(CBMC_UNWINDSET)
+
+GOTO_ANALYZER ?= goto-analyzer
+GOTO_CC ?= goto-cc
+GOTO_INSTRUMENT ?= goto-instrument
+HELPERDIR ?= $(SRCDIR)/.cbmc-batch
+
+INC += \
+	-I$(HELPERDIR)/. \
+	-I$(SRCDIR)/include/
+
+SIMPLIFY ?= 0
+SRCDIR ?= $(abspath ../../..)
+UNWIND ?= 0
+VIEWER ?= cbmc-viewer
+
+################################################################
+$(ENTRY)1.goto: $(ENTRY).c
```

Comment:
This should probably take dependencies as a dependency, so that the makefile knows to rebuild when needed.